README.md

```diff
@@ -1,3 +1,113 @@
-# cordova
+# Apache Cordova
 
-Apache Cordova
+- Mobile apps with HTML, CSS & JS
+- Target multiple platforms with one code base
+- Free and open source
+
+## Repositories
+
+A list of all repositories on GitHub: 
+
+### Platforms
+
+- https://github.com/apache/cordova-android
+- https://github.com/apache/cordova-ios
+- https://github.com/apache/cordova-windows
+- https://github.com/apache/cordova-browser
+- https://github.com/apache/cordova-osx
+- https://github.com/apache/cordova-test-platform
+
+#### Experimental or Unfinished Platforms
+
+- https://github.com/apache/cordova-electron
+
+#### Deprecated Platforms
+
+- https://github.com/apache/cordova-blackberry
+- https://github.com/apache/cordova-firefoxos
+- https://github.com/apache/cordova-ubuntu
+- https://github.com/apache/cordova-wp8
+- https://github.com/apache/cordova-tizen
+- https://github.com/apache/cordova-qt
+- https://github.com/apache/cordova-webos
+- https://github.com/apache/cordova-amazon-fireos
+- https://github.com/apache/cordova-wp7
+- https://github.com/apache/cordova-bada
+- https://github.com/apache/cordova-bada-wac
+
+### Tools
+
+- https://github.com/apache/cordova-js
+- https://github.com/apache/cordova-plugman
+- https://github.com/apache/cordova-lib
+- https://github.com/apache/cordova-cli
+- https://github.com/apache/cordova-coho
+- https://github.com/apache/cordova-medic
```

Comment:
I guess `cordova-medic` should go to "Unused / Outdated / Deprecated" section. It has been replaced by paramedic.

Reply:
done, thanks.